examples/l1-resources/stream-connection.ts

```diff
@@ -0,0 +1,60 @@
+import * as cdk from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import { CfnStreamConnection, CfnStreamConnectionPropsType } from 'awscdk-resources-mongodbatlas'
+import { env } from 'node:process';
+
+interface AtlasStackProps {
+	readonly projectId: string;
+	readonly profile: string;
+	readonly instanceName: string;
+	readonly connectionName: string;
+	readonly type: CfnStreamConnectionPropsType;
+}
+
+const app = new cdk.App();
+
+
+export class CdkTestStack extends cdk.Stack {
+	constructor(scope: Construct, id: string, props?: cdk.StackProps) {
+		super(scope, id, props);
+
+		const atlasProps = this.getContextProps();
+		const streamConnection = new CfnStreamConnection(this, "stream-connection-testing-stack", {
+			profile: atlasProps.profile,
+			instanceName: atlasProps.instanceName,
+			projectId: atlasProps.projectId,
+			connectionName: atlasProps.connectionName,
+			type: atlasProps.type
+		});
```

Comment:
I believe this would not be working example, depending on the type ("cluster" or "kafka") additional properties have to be provided. Would replicate cfn examples which has one for each type.

Reply:
You are right, this was outdated, now I updated it with cluster name which like I tested it in the manual QA